src/lib/sync.js

```diff
@@ -0,0 +1,22 @@
+var request = require('request')
+
+module.exports = function (opts, callback) {
+  request({
+    method: 'POST',
+    url: opts.api + 'sync',
+    json: true,
+    headers: {
+      Authorization: 'Bearer ' + opts.token
+    }
+  }, function (err, res, data) {
+    if (err) return callback(err)
+
+    if (!data.repos) {
+      var errorMessage = res.statusMessage
+      if (res.body.message) errorMessage += ': ' + res.body.message
+      return callback(new Error(errorMessage))
+    }
+
+    callback(null, data.repos)
+  })
+}
```

Comment:
The sync call was already duplicated in `login.js` https://github.com/greenkeeperio/greenkeeper/blob/master/src/login.js#L35.
Would be nice if we could get all three calls abstracted into one module (including the spinner).